ifrs17/Test/QueriesScenarioDataTest.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+    "metadata": {
+        "authors": [],
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"../Utils/Queries\"",
+                "\n#!import \"../Report/ReportConfigurationAndUtils\"",
+                "\n#!import \"TestData\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Initialization"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await DataSource.UpdateAsync( new[] { partitionReportingNode } );",
+                "\nawait DataSource.UpdateAsync( new[] { partition, previousPeriodPartition, partitionScenarioMTUP } );",
+                "\nawait DataSource.UpdateAsync( new[] { dt11 } );"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Import"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Data"
```

Comment:
These test could also contain interaction between primary and secondary import scope. 
At the moment I don't think QueryPartitionedDataAsync has any notion of the scope to query (it has only the partition). Correct?

Anyway, we plan to add a notion of import scope in #133. Can you add there a comment to add the above mention test case? thanks

Reply:
Ok, comment added.